### DISCUSSION TITLE
Cursor crossing screens when playing Valorant

### DISCUSSION THREAD

**Discussion by PurBlu** [input-leap/input-leap#2130] [Q&A] [answered]
Running server on my Windows machine and client on macOS.
Frequently playing games on my Windows machine. 
Most FPS games are just fine as they will lock the cursor at the center of the screen.
But for Valorant, even though I set the game running under 'Fullscreen' mode (not 'borderless Fullscreen'),
the cursor can still be moved during the game and therefore I have to kill all Inputleap related process every time I play it. Quite frustrated about this.

Would be really appreciated for a solution. 😁

**Comment by shymega** [comment, marked as answer]
Please open this as a bug, and provide the required details of software-related components - also, screen resolutions, game platform (Steam, etc), game resolution settings, and any mods used on games would be useful.
